.github/scripts/verifyPodfile.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
-declare -r GREEN='\033[0;32m'
+# This script accepts two arguments:
+# First argument (required): the branch to compare against for removals
+# Second argument (optional): `-v` which will output each branch's podspecs and the removals if they exist
```

Comment:
Let's just remove the second argument and always provide verbose output to keep the code simpler.